use_case_guide/real_time_dashboards.rst

```diff
@@ -314,9 +314,41 @@ like this:
 
 .. code-block:: sql
 
+  -- let's first use hll extension on a regular postgres table
+  CREATE TABLE http_request_local AS SELECT * FROM http_request;
```

Comment:
Removing the local part makes sense but I would be happy if we had some way to show that we would use `hll_add_agg` in the non-citus case.